app/src/main/java/ai/elimu/vitabu/ui/storybook/StoryBookActivity.kt

```diff
@@ -36,7 +36,7 @@ class StoryBookActivity : AppCompatActivity() {
         val chapterPagerAdapter = ChapterPagerAdapter(
             supportFragmentManager,
             storyBookChapters,
-            readingLevel,
+            readingLevel ?: ReadingLevel.LEVEL1,
```

Comment:
Can we leave the `null` value here? An uncategorized storybook will just appear at the top, which can help notify us that a book is missing reading level categorization.
```suggestion
            readingLevel,
```

> [!NOTE]
> In the future we will make it impossible to store a storybook in the webapp database without setting a reading level.

Reply:
@jo-elimu It's just a fallback value in case `readingLevel` cannot be found from intent's bundle data.

Reply:
@tuancoltech Okay, so the fallback to `LEVEL1` will never occur, I think is what you are saying?

Reply:
@jo-elimu The fallback can occur if and only if `readingLevel` is not present in `storybooks` API response, but even if it occurs, setting the fallback here won't impact anything since we're not using it in `CoverFragment`.